.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+Before submitting your PR, check that you've:
+
+- [ ] Documented new APIs with [Haddock markup](https://www.haskell.org/haddock/doc/html/index.html)
+- [ ] Added [`@since` declarations](http://haskell-haddock.readthedocs.io/en/latest/markup.html#since) to the Haddock
+
+After submitting your PR:
+
+- [ ] Update the Changelog.md file with a link to your PR
+- [ ] Bumped the version number if there isn't an `(unreleased)` on the Changelog
```

Comment:
This may be unnecessary - `persistent` likes to release quickly, and this line makes it relatively easy to get things out fast.

Reply:
`text` is not `persistent`. There is technically little value in releasing often as people only use versions bundled with GHC. (Stackage doesn't upgrade GHC bundled libs for example).

So `persistent` experience doesn't apply directly.

Reply:
Of course - happy to remove the line :)